diff.py

```diff
@@ -1857,6 +1865,7 @@ class Line:
     original: str
     normalized_original: str
     scorable_line: str
+    relocation_occured: bool
```

Comment:
occurred. or `has_symbol`; may be clearer. or maybe even make this a `symbol: Optional[str]`? then `imm_matches_everything` could test for it without needing to care about arch-specific syntax for relocations

Reply:
I started a new branch that worked like this method.. it took a good amount of reworking, but then the problem still remains we want to tell the difference between any any symbol from the relocation, and a symbol that is a catch anything symbol. And therefore we still need the logic to check for that somewhere.   And then also, doing this makes the method / process diverge more from the permutor code, which would take more reworking since it's not setup with an object that is accessed in both places.